modules/data.land/R/put.veg.module.R

```diff
@@ -49,24 +35,29 @@
     logger.severe(paste(fcn, "does not exist."))
   }
   
+  # NOTE : match_pft is now called in veg2model
```

Comment:
I'm not sure that's a good design decision -- any function that HAS TO be called within EVERY model-specific function really seems like a function that should be in the part of the workflow that PEcAn controls, not the model-specific code.